apps/src/blockly/customBlocks/googleBlockly/spritelabBlocks.js

```diff
@@ -22,22 +26,27 @@ export const blocks = {
         name: 'FLYOUT',
         isFlyoutVisible: true,
       });
-      block
-        .appendDummyInput('flyout_input')
-        .appendField(flyoutField, flyoutKey);
+
+      block.appendDummyInput(INPUTS.FLYOUT).appendField(flyoutField, flyoutKey);
+      // TODO: How can we confirm that this can be the default for all blocks?
+      if (block.getInput(INPUTS.STACK)) {
```

Comment:
We currently add mini toolboxes to the end of the block. This renders the mini-toolbox above the code/statement input/stack, which 1) is more similar to the current UI and 2) keeps you from having to scroll to the bottom to see the toolbox depending on how much code is in the behavior definition.

Reply:
Oh! I realized I responded to the wrong question. Mini-toolboxes render at the end of the block by default, which is the correct behavior for all call blocks. I was going to use an if statement like this here...
```
if (renderingInFunctionEditor) {
  block.moveInputBefore(INPUTS.FLYOUT, INPUTS.STACK);
}
```
However, @mikeharv called out that since definition blocks will all be hidden now, we could probably move the toolbox to the top for _any_ block with a stack (not impacting any of the existing toolboxes because they're not definition blocks). However, I'm leaning toward changing it back to an explicit condition related to the function editor because I think it's clearer/more explicit. 

@molly-moen Thoughts?

Reply:
Having it be explicitly linked to the function editor makes sense to me, that will be clearer.